config/diaspora.toml.example

```diff
@@ -0,0 +1,698 @@
+[configuration.environment.s3]
+
+#enable = true
+#key = "change_me"
+#secret = "change_me"
+#bucket = "my_photos"
+#region = "us-east-1"
+
+## Use max-age header on Amazon S3 resources (default=true).
+## When true, this allows locally cached images to be served for up to
+## one year. This can improve load speed and save requests to the image
+## host. Set to false to revert to browser defaults (usually less than
+## one year).
+#cache = true
+
+[configuration.environment.assets]
+
+## Serve static assets via the appserver (default=false).
+## This is highly discouraged for production use. Let your reverse
+## proxy/webserver do it by serving the files under public/ directly.
+#serve = false
+
+## Upload your assets to S3 (default=false).
+#upload = false
+
+## Specify an asset host. Ensure it does not have a trailing slash (/).
+#host = "http://cdn.example.org/diaspora"
+
+## Logger configuration
+[configuration.environment.logging]
+
+## Roll the application log on a daily basis (default=true).
+#logrotate.enable = true
+
+## The number of days to keep (default=7)
+#logrotate.days = 7
+
+## Enables the debug-logging for SQL (default=false)
+## This logs every SQL-statement!
+#debug.sql = true
+
+## Enables the federation-debug-log (default=false)
+## This logs all XMLs that are used for the federation
+#debug.federation = true
+
+## Settings affecting how ./script/server behaves.
+[configuration.server]
+
+## Where the appserver should listen to (default="unix:tmp/diaspora.sock")
+#listen = "unix:tmp/diaspora.sock"
+#listen = "unix:/run/diaspora/diaspora.sock"
+#listen = "127.0.0.1:3000"
+
+## Set the path for the PID file of the unicorn master process (default=tmp/pids/web.pid)
+#pid = "tmp/pids/web.pid"
+
+## Rails environment (default="development").
+## The environment in which the server should be started by default.
+## Change this to "production" if you wish to run a production environment.
+#rails_environment = "production"
+
+## Write unicorn stderr and stdout log.
+#stderr_log = "log/unicorn-stderr.log"
+#stdout_log = "log/unicorn-stdout.log"
+
+## Number of Unicorn worker processes (default=2).
+## Increase this if you have many users.
+#unicorn_worker = 2
+
+## Number of seconds before a request is aborted (default=90).
+## Increase if you get empty responses, or if large image uploads fail.
+## Decrease if you're under heavy load and don't care if some
+## requests fail.
+#unicorn_timeout = 90
+
+## Embed a Sidekiq worker inside the unicorn process (default=false).
+## Useful for minimal Heroku setups.
+#embed_sidekiq_worker = false
+
+## Number of Sidekiq worker processes (default=1).
+## In most cases it is better to
+## increase environment.sidekiq.concurrency instead!
+#sidekiq_workers = 1
+
+## Diaspora has an internal XMPP web-client. If you want to enable the chat
+## functionality or want to use a custom XMPP server, then you should edit
+## the following configuration.
+[configuration.chat]
```

Comment:
I doubt users that enabled the chat are quite getting that they could just migrate the chat options to the TOML (and how), people often struggle with the environment variable conversion that's explicitly described in the file already.

I wanted to avoid somebody on 0.7 doing the migration and then getting a feeling of being stuck/wasting time when they don't find the chat options anymore. 

Maybe we can mark it as deprecated?

Reply:
Yes, at least mark it as deprecated, I don't want people installing a new pod now to activate the chat.

And people on 0.7 with chat enabled could still just keep the old yaml until they upgrade to 0.8, or already disable the chat when migrating to toml. That's why I'm not sure if we really need it in the toml example